project/Dependencies.scala

```diff
@@ -62,24 +62,24 @@ object Dependencies {
     excludeGuava("com.google.apis"     % "google-api-services-sheets"       % "v4-rev20191001-1.30.3"),
     excludeGuava("com.google.apis"     % "google-api-services-cloudbilling" % "v1-rev20191005-1.30.3"),
     excludeGuava("com.google.apis"     % "google-api-services-pubsub"       % "v1-rev20191001-1.30.3"),
-    excludeGuava("com.google.auth"     % "google-auth-library-oauth2-http"  % "0.18.0"),
+    excludeGuava("com.google.auth"     % "google-auth-library-oauth2-http"  % "0.24.1"),
     excludeGuava("com.google.apis"     % "google-api-services-admin-directory"  % "directory_v1-rev110-1.25.0"),
 
-    "org.webjars.npm"                % "swagger-ui-dist"     % "3.35.0",
+    "org.webjars.npm"                % "swagger-ui-dist"     % "3.45.0",
     "org.webjars"                    % "webjars-locator"     % "0.40",
-    "com.pauldijou"                 %% "jwt-core"            % "3.1.0",
+    "com.github.jwt-scala"          %% "jwt-core"            % "7.1.1",
```

Comment:
This is one thing I think might benefit from further testing

Reply:
Cloud Integrations uses JWT, definitely check with them on this one